Blech, give us a relatively simple way to drop in "lib" or "lib64", as
appropriate.  Needs fixing to be more correct.

--- mrtg-2.15.0/bin/cfgmaker.lib64	2006-12-06 01:10:06.000000000 +0100
+++ mrtg-2.15.0/bin/cfgmaker	2006-12-06 12:35:16.000000000 +0100
@@ -44,7 +44,7 @@
 
 use FindBin;
 use lib "${FindBin::Bin}";
-use lib "${FindBin::Bin}${main::SL}..${main::SL}lib${main::SL}mrtg2";
+use lib "${FindBin::Bin}${main::SL}..${main::SL}@@lib@@${main::SL}mrtg2";
 
 use MRTG_lib "2.100015";
 use Getopt::Long;
--- mrtg-2.15.0/bin/mrtg-traffic-sum.lib64	2006-12-06 12:37:19.000000000 +0100
+++ mrtg-2.15.0/bin/mrtg-traffic-sum	2006-12-06 12:38:18.000000000 +0100
@@ -23,7 +23,7 @@
 
 use FindBin;
 use lib "${FindBin::Bin}";
-use lib "${FindBin::Bin}${main::SL}..${main::SL}lib${main::SL}mrtg2";
+use lib "${FindBin::Bin}${main::SL}..${main::SL}@@lib@@${main::SL}mrtg2";
 use MRTG_lib "2.090017";
 use POSIX qw(mktime);
 
--- mrtg-2.15.0/bin/indexmaker.lib64	2006-12-06 01:10:06.000000000 +0100
+++ mrtg-2.15.0/bin/indexmaker	2006-12-06 12:35:16.000000000 +0100
@@ -48,7 +48,7 @@
 
 use FindBin;
 use lib "${FindBin::Bin}";
-use lib "${FindBin::Bin}${main::SL}..${main::SL}lib${main::SL}mrtg2";
+use lib "${FindBin::Bin}${main::SL}..${main::SL}@@lib@@${main::SL}mrtg2";
 
 use MRTG_lib "2.100016";
 use Getopt::Long;
--- mrtg-2.15.0/bin/mrtg.lib64	2006-12-06 01:10:06.000000000 +0100
+++ mrtg-2.15.0/bin/mrtg	2006-12-06 12:35:16.000000000 +0100
@@ -81,7 +81,7 @@
 
 use FindBin;
 use lib "${FindBin::Bin}";
-use lib "${FindBin::Bin}${main::SL}..${main::SL}lib${main::SL}mrtg2";
+use lib "${FindBin::Bin}${main::SL}..${main::SL}@@lib@@${main::SL}mrtg2";
 use Getopt::Long;
 use Math::BigFloat;
 
